game.rb

```diff
@@ -0,0 +1,26 @@
+class Item
+  def can_be_archived
+    # Implementation of the can_be_archived method in the Item base class.
+    # In the future as member 1 I will place here the logic that determines if the item can be archived.
+  end
+end
```

Comment:
please review the Item Class implementation in the #35

Reply:
Done👍